server/src/main/java/io/spine/server/entity/storage/QueryParameters.java

```diff
@@ -75,6 +75,13 @@ public static Builder newBuilder() {
         return new Builder();
     }
 
+    public static Builder newBuilder(QueryParameters parameters) {
+        return new Builder()
+                .addAll(parameters)
```

Comment:
It seems to be misaligned. Maybe I am wrong. Please check.

Reply:
This alignment is performed by our IDEA config.